src/Util.js

```diff
@@ -13,22 +13,15 @@ function commonParams() {
     };
 }
 
-const TagRegex = /(refs\/)(tags\/M-staged-PR)(\d+)$/;
+const PrNumberRegex = /(.* \(#)(\d+)(\)$)/;
```

Comment:
If we do not need `.*`, please drop that part of the regex.

Reply:
Done.